### ISSUE TITLE
A11y_.NET Core_MAUI_Recipe App_Your Recipe_Keyboard : The "More options" button is not accessible by keyboard.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/maui#15664]
### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[MAUI]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG2.1.1;#DesktopApp;#Win11;#Benchmark;

### Actual:
The "More option" button present in the Right top corner is not accessible by keyboard. Keyboard focus does not moves to that control

### Expected:
The "More Option" button should be accessible by keyboard.

### User Impact:
Keyboard Users will face issue while accessing the controls/buttons.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. Tab till "My Recipes" and select.
3. Tab till any existing recipe and select.
4. "Recipe" Screen will open.
5. Tab till the "More option" button and observe the issue by hitting enter on it.

### Attachment:

[Recording-20230614_203048.webm](https://github.com/dotnet/maui/assets/85863895/4dfd8934-85e1-4b2c-ab60-8bb8fd183728)

### Link to public reproduction project repository

Test

### Version with bug

7.0.49

### Last version that worked well

6.0

### Affected platforms

Windows

### Affected platform versions

Windows 11 23H2

**Comment by Ashutoshsahoo164** [comment]
Hi @rachelkang The expected is While we navigate from Top to Bottom and also Bottom to top , the See more button should be accessible

**Comment by PoojaNamde** [comment]
#A11ypreview;

Added A11ypreview tag

**Comment by PoojaNamde** [comment]
@rachelkang Could you please close this issue as I am not getting close option.

#Regressed:09-21-23; #Closed;

Verified this issue in latest environment, issue is fixed hence closing the bug.
https://github.com/dotnet/maui/assets/90901519/7fc4b376-e80d-4fad-a5f0-8c8b58dc63ac
